### ISSUE TITLE
Implement custom keyboard navigation support

### ISSUE THREAD

**Issue by milesap** [fgpv-vpgf/fgpv-vpgf#808]
Implement keyboard navigation support similar to [Google Maps](https://www.google.ca/maps). Keyboard support should activate only when the map is focused (see related issue #52).

#### Overview
- Up/Down/Left/Right arrow keydown pans map north/south/west/east respectively by some arbitrary distance.
- + key zooms in one level if possible
- - key zooms out one level if possible
- Enter key selects point information from map center

#### Additional Information 
Existing ArcGIS keyboard navigation support only works if the users mouse is hovering over the map. If the user does not have a mouse or is hovering over a panel/navbar then keyboard support does not work.

See #52
